api4/user_test.go

```diff
@@ -1967,27 +1875,26 @@ func TestUpdateUserPassword(t *testing.T) {
 		t.Fatal("should have returned true")
 	}
 
-	_, resp = Client.Login(th.BasicUser.Email, adminSetPassword)
+	_, resp = th.Client.Login(th.BasicUser.Email, adminSetPassword)
 	CheckNoError(t, resp)
 }
 
 /*func TestResetPassword(t *testing.T) {
 	th := Setup().InitBasic()
 	defer th.TearDown()
-	Client := th.Client
-	Client.Logout()
+th.Client.Logout()
```

Comment:
Also, the indentation in this commented out test is a bit messed up. Maybe, we just uncomment it and add a `t.Skip()` to the start instead?